package.json

```diff
@@ -39,15 +39,14 @@
     "bs58check": "^2.1.1",
     "commander": "^2.14.1",
     "joi-browser": "^13.4.0",
-    "json-bigint": "github:davidyuk/json-bigint",
+    "json-bigint": "davidyuk/json-bigint",
```

Comment:
is there a way to change this before this release?
maybe fork under aeternity org.